my-app/src/App/pages/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import InfoProductCard from "./components/InfoProductCard";
+
+const ProductPage = () => {
```

Comment:
нужно объявлять тип для всех компонентов, функций, переменных и т.д. в данном случае должно быть:
```suggestion
const ProductPage: React.FC = () => {
```